repose-aggregator/src/docs/asciidoc/filters/saml-policy.adoc

```diff
@@ -15,15 +15,19 @@ It'll attempt to locate a policy to use for the translation based on the asserti
 
 == Prerequisites & Postconditions
 === Required headers
-`Content-Type` - Needs a value of `application/xml`
+`Content-Type` - Needs a value of `application/x-www-form-urlencoded`
 
 === Required preceding filters
-If using the https://en.wikipedia.org/wiki/SAML_2.0#HTTP_POST_Binding[post form binding], you'll need to use the <<saml-binding.adoc#,SAML Binding Translation Filter>> to decode the request first.
+None
 
 === Request headers created
+`Content-Type` - Will be changed to `application/xml`
+`Content-Length` - Will be removed if present
+`Transfer-Encoding` - Will be set to `chunked`
```

Comment:
Should we put these in a table or maybe a list?  Right now they're in a paragraph, and it doesn't look that great.  I'm not sure if you'd also want to update the "Required headers" section for consistency, although it's only one header under that section.